client/selfrepair/tests/test_normandy_driver.js

```diff
@@ -36,21 +39,106 @@ describe('Normandy Driver', () => {
     });
   });
 
+  describe('saveHeartbeatFlow', () => {
+    const driver = new NormandyDriver();
+
+    afterEach(() => {
+      fetchMock.restore();
```

Comment:
We should have something like `expect(fetchMock.calls().unmatched).toEqual([])`, to check that the tests didn't make any errant fetch calls.